man/concepts_descendants.Rd

```diff
@@ -0,0 +1,63 @@
+% Generated by roxygen2: do not edit by hand
+% Please edit documentation in R/wrapper.R
+\name{concepts_descendants}
+\alias{concepts_descendants}
+\title{Fetch descendants of one or more concepts}
+\usage{
+concepts_descendants(
+  conceptIds,
+  direct_descendants = FALSE,
+  activeFilter = TRUE,
+  encoding = "UTF-8",
+  silent = FALSE,
+  ...
+)
+}
+\arguments{
+\item{conceptIds}{a character vector of concept identifiers}
+
+\item{direct_descendants}{a logical vector indicating whether to fetch
+direct descendants (children) of the \code{conceptIds} exclusively or all
+descendants (including children). The default is \code{FALSE}. If a single
+value is provided, it will be recycled.}
+
+\item{activeFilter}{a logical vector indicating whether to fetch active
+descendant concepts exclusively. The default is \code{TRUE}. If a single
+value is provided, it will be recycled.}
+
+\item{encoding}{HTTP charset parameter to use (default is \code{"UTF-8"})}
+
+\item{silent}{whether to hide progress bar. Default is \code{FALSE}}
+
+\item{...}{other valid arguments to function \code{\link{api_concepts}},
+for instance \code{endpoint}, \code{branch} or \code{limit}.}
+}
+\value{
+a named list of data frames
+}
+\description{
+This function is a wrapper of \code{\link{api_concepts}} that
+fetches descendants of one or several concept identifiers using full SNOMED CT
+inference. The search can be restricted to children concepts using the
+\code{direct_descendants} argument.
```

Comment:
this function will return a nested list, with one entry for each parent conceptId provided in input.
So in case of overlap the descendants will appear as many times as there are parents.